src/main/java/io/fabric8/maven/docker/util/AnsiLogger.java

```diff
@@ -28,7 +28,7 @@
     private final String prefix;
     private final boolean batchMode;
 
-    private boolean verbose;
+    private String verbose = "build";
```

Comment:
better call it "verboseModes" and maybe make it an array of LogVerboseCategory enums ? see below.